WAVLTree/src/WAVLTree.java

```diff
@@ -121,7 +140,11 @@ public int size()
     */
    public WAVLNode getRoot()
    {
+       if (this.empty()) {
```

Comment:
Once @Onoam implements the external node, we should be able to just use `return this.root`  or equivalently, `return root`